app/views/layouts/_candidate_header.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if candidate_signed_in? %>
+  <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+    <li class="govuk-header__navigation-item">
+      <b><%= current_candidate.email_address %></b>
+    </li>
+
+    <li class="govuk-header__navigation-item">
+      <%= link_to 'Sign out', candidate_interface_sign_out_path, class: 'govuk-header__link' %>
+    </li>
+  </ul>
+<% end %>
```

Comment:
should we add a test for it? 
if `candidate_signed_in` the navigation and `Sign out` should be visible and working

Reply:
I think that's already tested in tests like https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/spec/system/candidate_interface/candidate_account_spec.rb.